bin/ci/patch_plugin_gemfile_lock

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
```

Comment:
hmm, this is a new file... where is it being called from?

Reply:
:point_up: https://github.com/ManageIQ/manageiq/pull/22075#issue-1349718390
> It will have to be called by the bin/before_install script in each plugin.

Reply:
Ah, cool... did you try a cross repo with a plugin to see how it works?  I really want to see what the before/after or the difference and compare it with what happens now. It sounds like this is plugging a clear hole in our testing but it's not obvious.

Reply:
> did you try a cross repo with a plugin to see how it works

Haha this was the reason for my comment about the GITHUB_REPOSITORY being wrong, we can't test this on cross-repo easily.

Reply:
What I did instead to test this was copy the Gemfile.lock.release from `oparin` over to an autosde branch, force a downgrade of a gem (autosde_openapi_client from .34 -> .33) and check that the older version is used (aka it is using what is in the Gemfile.lock rather than pulling the latest).

https://github.com/ManageIQ/manageiq-providers-autosde/pull/168/files#diff-14d541c92b03a497ca2efd069b0a5a0fbc47487e0d8a2d366933d6ee44103a4cR30 (NOTE copy from `$gem_root` instead of `$spec_manageiq` because of the cross-repo issue)

Here is it using the `1.1.33` versus the latest `1.1.34`, https://github.com/ManageIQ/manageiq-providers-autosde/runs/7999317052?check_suite_focus=true#step:6:493